### PR TITLE
Issue 1477/layman login iframe

### PR DESCRIPTION
**Pull request by raitisbe**

### CODE REVIEW

**Review by DailisLangovskis — APPROVED** [review]

File: projects/hslayers/src/common/layman/layman-current-user.component.ts [hunk 1]
```diff
@@ -40,14 +40,19 @@ export class HsLaymanCurrentUserComponent {
     return this.endpoint.url + '/authn/oauth2-liferay/login';
   }
 
+  /**
+   * @description Periodically poll layman client endpoint for auth change.
```
Comment (jmacura) [inline projects/hslayers/src/common/layman/layman-current-user.component.ts:44]:
```suggestion
   * Periodically poll layman client endpoint for auth change.
```

File: projects/hslayers/src/common/layman/layman.service.ts [hunk 2]
```diff
@@ -9,38 +9,43 @@ export class HsCommonLaymanService {
   authChange = new Subject();
   constructor(private $http: HttpClient) {}
 
-  async getCurrentUser(endpoint): Promise<any> {
+  /**
+   * @description Monitor if authorization state has changed and
```
Comment (jmacura) [inline projects/hslayers/src/common/layman/layman.service.ts:13]:
```suggestion
   *  Monitor if authorization state has changed and
```

File: projects/hslayers/src/common/layman/layman.service.ts [hunk 3]
```diff
@@ -9,38 +9,43 @@ export class HsCommonLaymanService {
...
+   * return true and broadcast authChange event if so .
+   * @param endpoint Endpoint definition - usually Layman
```
Comment (jmacura) [inline projects/hslayers/src/common/layman/layman.service.ts:15]:
```suggestion
   * @param endpoint - Endpoint definition - usually Layman
```

**Review by jmacura — COMMENTED** [review]
Little TSDoc fixes

**Pull request merged**